src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/Encoding/BlobEncoders.cs

```diff
@@ -558,7 +558,7 @@ public void SystemType(string serializedTypeName)
             String(serializedTypeName);
```

Comment:
Above row 551 `serializedTypeName` is nullable

Reply:
Despite what the docs say, we are throwing if `serializedTypeName` is null or empty. Doesn't that mean we should annotate as non-nullable? /cc @stephentoub